lib/node_modules/@stdlib/string/base/slice-code-points/test/test.js

```diff
@@ -0,0 +1,174 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var tape = require( 'tape' );
+var sliceCodePoints = require( './../lib/main.js' );
+
+
+// TESTS //
+
+tape( 'main export is a function', function test( t ) {
+	t.ok( true, __filename );
+	t.strictEqual( typeof sliceCodePoints, 'function', 'main export is a function' );
+	t.end();
+});
+
+tape( 'the function has an arity of 3', function test( t ) {
+	t.strictEqual( sliceCodePoints.length, 3, 'the function has an arity of 3' );
+	t.end();
+});
+
+tape( 'the function returns an empty string if provided an empty string', function test( t ) {
+	t.strictEqual( sliceCodePoints( '', 1, 2 ), '', 'returns expected value' );
+	t.strictEqual( sliceCodePoints( '', 2, 3 ), '', 'returns expected value' );
+	t.strictEqual( sliceCodePoints( '', 3, 4 ), '', 'returns expected value' );
+	t.end();
+});
+
+tape( 'the function throws an error if provided an unsupported number of arguments', function test( t ) {
```

Comment:
This test is no longer appropriate given that we are not explicitly asserting whether the correct number of arguments is provided.